docs/configuration/optional-settings.md

```diff
@@ -135,6 +135,22 @@ Enforcement of unique IP space can be toggled on a per-VRF basis. To enforce uni
 
 ---
 
+## GITHUB_REPOSITORY_API
```

Comment:
Let's avoid using `GITHUB` in the parameter names: We might extend this at some point in the future to work with other services, and it would be nice not to have to rename them.

Maybe `UPDATE_REPO_URL` and `UPDATE_CACHE_TIMEOUT`? Open to other suggestions.